<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletServiceBaseSettingsUi</class>
 <widget class="QWidget" name="AppletServiceBaseSettingsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Applet Service Base Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_ServiceTitleLabel">
     <property name="text">
      <string>Service Title</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="InformationWidget" name="m_InfoWidget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="m_ApplyButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Apply Changes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_PermissionLabel">
     <property name="text">
      <string>Permission Level Required To Use Service</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PermissionWidget" name="m_PermissionWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_ServiceImageLabel">
     <property name="text">
      <string>Optional Image For Detail Information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailEditWidget" name="m_ThumbnailEditWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_UrlDescriptionLabel">
     <property name="text">
      <string>Service URL or IP Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_UrlEdit">
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_UserDisplayedNameLabel">
     <property name="text">
      <string>Name Of Service ( Optional For Detail Information )
( maximum 63 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_NameEdit">
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_DescriptionLabel">
     <property name="text">
      <string>Description Of Service ( Optional For Detail Information )
( maximum 511 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="m_DescriptionEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LogWidget" name="m_LogWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThumbnailEditWidget</class>
   <extends>QWidget</extends>
   <header>ThumbnailEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PermissionWidget</class>
   <extends>QWidget</extends>
   <header>PermissionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InformationWidget</class>
   <extends>QWidget</extends>
   <header>InformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>LogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gotvcommon.qrc"/>
 </resources>
 <connections/>
</ui>
